components/HeadComponent.js

```diff
@@ -0,0 +1,34 @@
+import Head from "next/head";
+import styled from "styled-components";
+
+export default function HeadComponent() {
+  return (
+    <>
+      <Head>
+        <title>Fork yeah!</title>;
+      </Head>
+      <StyledHeadContainer>
+        <h1>Fork yeah!</h1>
+      </StyledHeadContainer>
+    </>
+  );
+}
+
+const StyledHeadContainer = styled.div`
```

Comment:
```suggestion
const StyledHeadContainer = styled.header`
```
You can use a more semantic html element here :)